backend/services/geminiQueryService.js

```diff
@@ -0,0 +1,50 @@
+const fetch = require('node-fetch');
+const logger = require('../utils/logger.js');
+const Config = require('config');
+
+async function fetchRestaurantRecommendations(location) {
+  const apiUrl = Config.get('application.gemini.api.url');
+  const apiKey = 'AIzaSyCa-FGEff49LSRM3ITCnJ6LyhRRWFE9d3s';
```

Comment:
you can follow the logic in my other pr to put all configs/creds on github actions and load them into .env file.

local config thats used for testing can br hardcoded into the .env file and committed. They will be overwritten on at CI stage